llvm/lib/Transforms/InstCombine/InstCombineCalls.cpp

```diff
@@ -1500,7 +1500,7 @@ Instruction *InstCombinerImpl::visitCallInst(CallInst &CI) {
   // Don't try to simplify calls without uses. It will not do anything useful,
   // but will result in the following folds being skipped.
   if (!CI.use_empty()) {
-    SmallVector<Value *, 4> Args;
+    SmallVector<Value *, 8> Args;
```

Comment:
Can this be rewritten as

```
SmallVector<Value *, 8> Args(CI.arg_begin(), CI.arg_end());
```

Reply:
Yes, thank you for bringing this up.  It can actually be even shorter:

```
SmallVector<Value *, 8> Args(CI.args());
```

I'll post a follow-up PR.